### ISSUE TITLE
Profiler for Soil metagenomes

### ISSUE THREAD

**Issue by EorgeKit** [ngless-toolkit/ng-meta-profiler#3]
#### What's the recommendation when dealing with soil metagenomes since there is no particular predefined profiler for it or any other metagenome that has no predefined profiler for that matter?

**Comment by luispedro** [comment]
Hi @EorgeKit,

For soil in particular, I would really recommend going through the assembly+MAG building route (i.e., a _de novo_ approach) rather than profiling against existing databases.

**Comment by EorgeKit** [comment]
Thanks @luispedro 
t seems as though ngless is not customized for denovo , or I am missing something?

**Comment by luispedro** [comment]
Yes, there is more support for reference-based approaches, but you can assemble too.

You can ask more open ended questions at

https://groups.google.com/g/ngless
